samples/core/pom.xml

```diff
@@ -213,6 +213,7 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
+      <scope>provided</scope>
```

Comment:
I already moved the files `SpringBootApp.java` and `SpringBootBatchApp.java` to the server project. For testing I changed the `scope` of `spring-boot-starter-web` to `test`. Compilation fails, because `class SetCharacterEncodingFilter` is not available without this dependency (the according bean is intantiated in [`WebConfig`](https://github.com/oasp/oasp4j/blob/develop/samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/WebConfig.java#L75-L84)). 

I propose to move `WebConfig` (and in fact any execution related stuff - whatever this means) to the server.
@hohwille  What do you think?